include/rocksdb/options.h

```diff
@@ -57,6 +57,21 @@ class FileSystem;
 struct Options;
 struct DbPath;
 
+// To indicate the file types
+enum FileType {
+  kWalFile,
+  kDBLockFile,
+  kTableFile,
+  kDescriptorFile,
+  kCurrentFile,
+  kTempFile,
+  kInfoLogFile,  // Either the current one, or an old one
+  kMetaDatabase,
```

Comment:
What is a "kMetaDatabase"?  Is this something that has not been developed or is it dead code?  Not sure if this belongs in the public API...